Обновив файл order-food.c

PravdyvetsVladyslav opened the pull request:
Об'єднав інші частини коду в цей файл та дописав частину коду, яка відповідає за реалізацію меню і обробку вибору користувача в меню.

PravdyvetsVladyslav committed:
Обновив файл order-food.c

Об'єднав інші частини коду в цей файл та дописав частину коду, яка відповідає за реалізацію меню і обробку вибору користувача в меню.

YaroslavPonzel commented on order-food.c:
```diff
@@ -25,4 +25,181 @@ User createNewUser(int userId, const char* username) {
+		
+		switch (choice) {
+            case 1:
+                printf("\nMain Course Items:\n");
```
дуже багато повторюваного коду в кейсах. Винесіть це в окрему функцію і вже її використовуйте

PravdyvetsVladyslav committed:
Оновив файл order-food.c

Переробив код згідно коментаря: виніс повторюваний код, що знаходився в switch(), в окремі функції

PravdyvetsVladyslav reviewed (commented):
Виправив код згідно коментаря

YaroslavPonzel commented:
> Виправив код згідно коментаря

виглядає набагато краще
